hparams/gan.yml

```diff
@@ -1,16 +1,16 @@
----
 hparams:
+  name: 'gan'
   plot: False
   epochs: 100
-  steps_per_epoch: 1000
+  steps: 1000
   sample_rate: 16000
   batch_size: 32
   buffer_size: 1000
   latent_size: 100
   generator_scale: 128
   gen_lr: 0.0001
   disc_lr: 0.0004
-  log_amin: 1e-5
+  log_amin: 0.00001
   num_examples: 16
   save_dir: './'
```

Comment:
Shouldn't that be `ckpts/gan`? Either way having a slash here but not in the VAE config bothers me.